examples/acoustics_2d_mapped/setplot.py

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/env python
```

Comment:
Why is this in the PR?

Reply:
I must have went in to view the contents and pressed space.